### DISCUSSION TITLE
Password required to volunteer for tasks

### DISCUSSION THREAD

**Discussion by TimB111** [liedekef/events-made-easy#35] [General]
Would it be possible to have a password for volunteers to sign up for each task? For our events we only want certain people who we have vetted to sign up to run an event. Ideally, when creating the list of tasks for an event, it would give most flexibility if there could be a unique password for each task. Then we could give the password to selected individuals and allow them to volunteer for specific events.

I can see that there's an option to require WP membership so that only logged-in users can sign up for tasks, but this still allows all users to volunteer, which means inexperienced people could take charge of an event and we want to avoid this.

**Comment by liedekef** [comment]
Tasks are grouped together, so a password per task is not possible. I'll think about this for a while, but there's no easy solution that fits every case ... I try also to avoid to treat volunteer tasks as event bookings.

**Comment by liedekef** [comment]
I checked the code a bit, adding approval would be possible but will take quite some time coding ...

**Reply by TimB111** [reply]
If the tasks are grouped together then one password per event to volunteer would work.

Or an approval process would also work well.

**Reply by liedekef** [reply]
Feel free to open a feature request in the Issues section (and feel more free to make a donation to make it happen sooner :-) )

**Reply by TimB111** [reply]
Thank you, I will do!
